src/settings/settings.service.ts

```diff
@@ -0,0 +1,57 @@
+import { HttpException, HttpStatus, Injectable } from '@nestjs/common';
+import { PrismaService } from 'src/prisma.service';
+import { UpdateUserDto } from './dto/update-user.dto';
+import { UpdatePasswordDto } from './dto/update-password.dto';
+import * as bcrypt from 'bcryptjs';
+
+@Injectable()
+export class SettingsService {
+  constructor(private prisma: PrismaService) {}
+
+  async updateUser(userId: number, updateUserDto: UpdateUserDto) {
+    try {
+      const updatedUser = await this.prisma.user.update({
+        where: {
+          id: userId,
+        },
+        data: {
+          fullName: updateUserDto.fullName,
+          username: updateUserDto.username,
+        },
+      });
+
+      return updatedUser;
```

Comment:
![image](https://github.com/user-attachments/assets/aab32de5-c0fb-4321-a1eb-5dbbcb983124)

허걱스 비밀번호도 같이 응답으로 넘어와서 password 부분은 제외하고 반환해도 좋을 것 같네유

```suggestion
    const { password, ...result } = updatedUser;
    return result;
```

Reply:
감사합니당! 👍